gpiozero/exc.py

```diff
@@ -192,7 +192,10 @@ class SPIWarning(GPIOZeroWarning):
     "Base class for warnings related to the SPI implementation"
 
 class SPISoftwareFallback(SPIWarning):
-    "Warning raised when falling back to the software implementation"
+    "Warning raised when falling back to the SPI software implementation"
+
+class PWMSoftwareFallback(SPIWarning):
```

Comment:
I'm not sure if it (logically) makes sense for `PWMSoftwareFallback` to be a subclass of `SPIWarning` ? :man_shrugging:

Reply:
True